feconf.py

```diff
@@ -83,6 +83,10 @@
     'TextInput': {
```

Comment:
That will break a lot of backend tests. Many tests internally rely on these values to test classifier_services functionality.